opentelemetry/chart/values.yaml

```diff
@@ -4,6 +4,28 @@ global:
   ## common labels to apply to all resources
   commonLabels: {}
 
+## C default rules for monitoring the cluster
+##
+prometheusRules:
+
+  ## Enables PrometheusRule resources to be created
+  create: true
+
+  disabled: {}
+    # ReconcileErrors: false
+
+  ## Labels for PrometheusRules
+  labels: {}
+
+  ## Annotations for PrometheusRules
+  annotations: {}
+
+  ## Additional labels for PrometheusRule alerts
+  ## This is useful for adding additional labels such as "support_group" or "service" for the routing of alerts to each rule
+  additionalRuleLabels:
+    # support_group: support
+    # service: my-service
```

Comment:
Please mention the new options in the `plugindefintion.yaml`. This helps to understand where to put the `support_group` and `service` labels.